changelog/8.6.0.yaml

```
@@ -0,0 +1,283 @@
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1856
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1853
+      timestamp: 1669845868
+      file:
+        name: 1669845868-add-agent-logging-level-to-fleet-configuration.yaml
+        checksum: 33cbf01b52f8cf30971d84fadbba10163197fe61
+    - kind: feature
+      summary: Ensure unique input ID when variable substition occurs from a dynamic provider
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1866
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1751
+      timestamp: 1669928791
+      file:
+        name: 1669928791-Ensure-unique-input-ID-when-variable-substition-occurs-from-a-dynamic-provider.yaml
+        checksum: c057e4c3420b9549dc31c8e13e58ad3992a8eb49
+    - kind: bug-fix
+      summary: Elastic-agent will use local port when running fleet-server
+      description: The elastic-agent will now use the 8221 locally bound port when running fleet-server instead of the external port (8220).
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1867
+      issue:
+        - https://github.com/elastic/ingest-dev/issues/1394
+      timestamp: 1669929902
+      file:
+        name: 1669929902-fix-local-fleet-server-port.yaml
+        checksum: 94f37fd1ff74bd320ec25cfcf2e2a0890eda494a
+    - kind: feature
+      summary: Remove inputs when all streams are removed
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1869
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1868
+      timestamp: 1669940990
+      file:
+        name: 1669940990-Remove-inputs-when-all-streams-are-removed.yaml
+        checksum: 3fcf26a5bbb988db85c0c5923d4bd2284dc95866
+    - kind: feature
+      summary: No longer restart Elastic Agent on log level change
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1914
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1896
+      timestamp: 1670518841
+      file:
+        name: 1670518841-No-longer-restart-Elastic-Agent-on-log-level-change.yaml
+        checksum: 12b34904cd47182a45e6e113518ef91ec718ebac
+    - kind: breaking-change
+      summary: Each input in an agent policy must have a unique ID.
+      description: |
+        Each input in an agent policy must have a unique ID, like "id: my-unique-input-id".
+        This only affects standalone agents. Unique IDs are automatically generated in
+        agent policies managed by Fleet.
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1994
+      issue: []
+      timestamp: 1671748292
+      file:
+        name: 1671748292-input-ids-are-required.yaml
+        checksum: 3ca6d7c10ce16c72c74b67b18916407ad09459fd
+    - kind: breaking-change
```

```suggestion
    - kind: upgrade
```

Not a breaking change for users of the Elastic agent, only developers.